src/components/nav-bar/DesktopContent.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AppButton from '../buttons/bases/AppButton';
+import HubblrImage from '../image-components/HubblrImage';
+
+const DesktopContent = ({ desktopLeftContent, desktopRightContent }) => {
+  return (
+    <>
+      <div className="flex items-center">
+        <div className="w-1/3">{desktopLeftContent}</div>
+        <div className="w-1/3 self-center flex justify-center">
+          <AppButton
+            className="button-dark"
+            onClick={(e) => {
+              e.preventDefault();
+              window.location.href = '/';
```

Comment:
Bei internen redirects sollte Gatsby's navigate benutzt werden. S. z.B. https://stackoverflow.com/questions/47583856/window-location-href-vs-history-pushstate-which-to-use (nicht ganz das gleiche aber erklärt das Hauptproblem / das Vermeiden von unnötigen http requests).

Reply:
Done - kannst du mal schauen, ob ich das richtig gemacht habe?
war mir nicht ganz sicher, wo <Link> hin muss mit dem onClick und so